src/component/BreadCrumb.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Breadcrumbs, Typography} from "@mui/material";
+
+function BreadCrumb({data}) {
+  const values = Object.values(data);
+  return (
+    <div>
+      <Breadcrumbs
+        aria-label="breadcrumbs"
+        sx={{
+          Weight: 400,
+          fontSize: "1.5rem",
+          Size: "1.5rem",
+          lineHeight: "2.25rem",
+          Letter: "0.25 px",
+        }}
+      >
+        {[...values].map((item) => (
+          <Typography
+            key={item.length}
+            fontSize="inherit"
+            Font="Roboto"
+            Align="Center"
+            color={values.indexOf(item) === values.length - 1 && "text.primary"}
```

Comment:
above map function already provides current index as the third arg, we can use -

```
values.length && values.map((item, index) => (
<Typography
            key={item.length}
            fontSize="inherit"
            Font="Roboto"
            Align="Center"
            color={ index=== values.length--1 ? "text.primary" : ""}
 ```

Reply:
updated map to use index